src/modules/Generator.jl

```diff
@@ -0,0 +1,114 @@
+"""
+Provides the functions related to generating documentation stubs.
+"""
+module Generator
+
+"""
+    savefile(f, root, filename)
+
+Attempts to save a file at `\$(root)/\$(filename)`. `f` will be called with file
+stream (see [`open`](http://docs.julialang.org/en/latest/stdlib/io-network/#Base.open)).
+
+`filename` can also be a file in a subdirectory (e.g. `src/index.md`), and then
+then subdirectories will be created automatically.
+"""
+function savefile(f, root, filename)
+    filepath = joinpath(root, filename)
+    if ispath(filepath) error("$(filepath) already exists") end
+    info("Generating $filename at $filepath")
+    mkpath(dirname(filepath))
+    open(f,filepath,"w")
+end
+
+"""
+Contents of the default `make.jl` file.
+"""
+function make(pkgname)
+    """
+    using Documenter
+    using $(pkgname)
+
+    makedocs(
+        modules = [$(pkgname)]
+    )
+
+    # Documenter can also automatically deploy documentation to gh-pages.
+    # See "Hosting Documentation" and deploydocs() in the Documenter manual
+    # for more information.
+    #=deploydocs(
+        repo = "<repository url>"
+    )=#
+    """
```

Comment:
I actually specifically went with the multilines so that it would be easier for the user to comment it in. I.e. this tries to be a commented out code block, not just a code example. For three lines it doesn't matter much of course, but would you still go with normal ones?